core/jobs/job_utils_test.py

```diff
@@ -198,6 +198,9 @@ def test_get_beam_key_from_ndb_key(self):
             namespace=self.namespace
         )
 
+        # We use private _from_ds_key here because it provides a functionality
```

Comment:
...provides functionality for obtaining an NDB key from a Beam key, and writing ...

Reply:
Done!